src/components/ec-date-range-filter/ec-date-range-filter.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <ec-filter-popover
+    :label="label"
+    :number-of-selected-filters="numberOfSelectedFilters"
+    :popover-options="popoverOptions"
+  >
+    <template #filter>
+      <div
+        class="ec-date-range-filter"
+        data-test="ec-date-range-filter"
+      >
+        <div
+          class="ec-date-range-filter__inputs-wrapper"
+          data-test="ec-date-range-filter__inputs-wrapper"
+        >
+          <ec-input-field
+            v-model="fromValueDate"
+            class="ec-date-range-filter__from-label"
+            type="date"
+            placeholder="DD/MM/YYYY"
```

Comment:
if we're not supporting yet different locales for our date fields the least we can do is being consistent how we're showing the placeholder (see capitalization) https://bitbucket.org/fxsolutions/eburyonline/src/ba86b02b8b4b249a806b314b41682158588804c5/frontend/src/views/trade-finance/submitted-requests/create-request/components/form/form.vue#lines-265

is the time to start thinking about a date field component? it has commonalities such error messages